<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CMindMapDtTaskTool</class>
 <widget class="QWidget" name="CMindMapDtTaskTool">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>247</width>
    <height>126</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Task</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="0,1,0">
   <property name="spacing">
    <number>4</number>
   </property>
   <property name="leftMargin">
    <number>4</number>
   </property>
   <property name="topMargin">
    <number>4</number>
   </property>
   <property name="rightMargin">
    <number>4</number>
   </property>
   <property name="bottomMargin">
    <number>4</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="spacing">
      <number>2</number>
     </property>
     <item>
      <widget class="QComboBox" name="m_cbPriority">
       <item>
        <property name="text">
         <string>--Priority--</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Priority 0</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Priority 1</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Priority 2</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Priority 3</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Priority 4</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Priority 5</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Priority 6</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="m_cbProgress">
       <property name="sizeAdjustPolicy">
        <enum>QComboBox::AdjustToContents</enum>
       </property>
       <item>
        <property name="text">
         <string>-- Status --</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>0%</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>25%</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>50%</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>75%</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>100%</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QFormLayout" name="formLayout">
     <property name="horizontalSpacing">
      <number>2</number>
     </property>
     <property name="verticalSpacing">
      <number>4</number>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Due Date:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1,0">
       <property name="spacing">
        <number>2</number>
       </property>
       <item>
        <widget class="QLineEdit" name="m_dueDate">
         <property name="readOnly">
          <bool>true</bool>
         </property>
         <property name="placeholderText">
          <string>--Not Set--</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="m_btnShowDateEditor">
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset resource="../mindstorm.qrc">
           <normaloff>:/mindstorm/icons/ui-calendar.png</normaloff>:/mindstorm/icons/ui-calendar.png</iconset>
         </property>
         <property name="autoRaise">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Assigned To:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="m_cbAssignedTo">
       <property name="editable">
        <bool>false</bool>
       </property>
       <item>
        <property name="text">
         <string>--Not Set--</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Customize...</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,1">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QToolButton" name="m_btnClear">
       <property name="toolTip">
        <string>Clear Task</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="../mindstorm.qrc">
         <normaloff>:/mindstorm/icons/ui-delete.png</normaloff>:/mindstorm/icons/ui-delete.png</iconset>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QWidget" name="widget_2" native="true"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../mindstorm.qrc"/>
  <include location="../mindstorm.qrc"/>
 </resources>
 <connections/>
</ui>
